base_external_dbsource/README.rst

```diff
@@ -46,15 +45,20 @@ To use this module:
 Known issues / Roadmap
 ======================
 
+* Find a way to remove or default the CA certs dir
+* Add concept of multiple connection strings for one source (multiple nodes)
+* Add a ConnectionEnvironment that allows for the reuse of connections
+* Message box should be displayed instead of error in ``connection_test``
+* Remove old api compatibility layers (v11)
+* Implement better CRUD handling
+
+
 Bug Tracker
 ===========
 
 Bugs are tracked on `GitHub Issues <https://github.com/OCA/server-tools/issues>`_.
 In case of trouble, please check there if your issue has already been reported.
-If you spotted it first, help us smashing it by providing a detailed and welcomed feedback `here <https://github.com/OCA/
-server-tools/issues/new?body=module:%20
-base_external_dbsource%0Aversion:%20
-9.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback.
```

Comment:
This is part of the template, and while I do agree the wording needs to be changed, it is out of scope here. Proposing a change to the template is something I've been meaning to do, you're welcome to as well - https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst